### DISCUSSION TITLE
Hevelius Mark Crossley's work

### DISCUSSION THREAD

**Discussion by DjokerFab** [Stellarium/stellarium#3469] [General]
Hi everyone,
I've just discovered on the web today Mark Crossley's work on adding Hevelius figures to Stellarium : great job ! I tried to send him an email but the address seems unfonctionnal, that's why I write here.
I'm very interested by ancient astronomical maps and especially the work of Hevelius. As I'm working on these maps and figuring out the constellations positioning on the night sky, I couldn't help myself but to notice some differences in the way Mark placed some drawings on the night sky, and some gaps considering certain stars, comparing to Hevelius original star maps/drawings (the ones where the main stars appear in the drawings).
I don't know if you will read this, Mark, but I'd be interested to share about your methods to add the drawings on Stellarium, i imagine it wasn't a piece of cake ! Did you place each drawing separately ? Did you have to deform the drawings ? Did you set the stellarium time in the 17th century when you did it ? etc... My observations are only details considering the whole picture. Once again, great job ! 
Best regards and good night everyone ;)
Fabien, from France.

**Comment by gzotti** [comment]
Yes, this work is very good! Yes, the figures have been isolated. Each figure of "constellation art" is linked with three stars only, so there are unavoidable distortions. The process is described in the Stellarium User Guide.

**Reply by DjokerFab** [reply]
Thanks for the answer, gzotti, I get the idea and it might explain that some stars are not aligned with the original drawings. I'm not familiar with the User Guide, I'll look for it to understand the whole process ;) Apart from the distorsions, I noticed one reverse drawing (Triangulum australe), but again, it is only a detail.

**Comment by Atque** [comment]
Why isn't this sky culture in the main distribution? It's very well-made, and since Hevelius has been dead for well over 300 years, I can't imagine legal problems.

**Reply by alex-w** [reply]
Unfortunately, according to the HubbleSOURCE site (the source of artwork for Mark Crossley's SC), these images can only be used “in planetarium shows, exhibits, videos, and other non-commercial applications.”, so the sky culture can’t be licensed under a free license and distributed together with Stellarium’s main distribution.

**Reply by gzotti** [reply]
Actually we also want to split skycultures from the main distribution at some time, like we also have downloadable landscapes. Many users are not interested in anything but the default or even Rey figures, so "why add another 10MB of bloat"? Download and installing takes a minute or two for those interested. An open problem is still how to add translation for those add-ons.

But I also agree, after unhappy events last year, unknown- or non-CC licensed skycultures can no longer be packed in the default set.
